app/src/main/java/com/rockthevote/grommet/data/db/dao/SessionDao.kt

```diff
@@ -15,9 +17,12 @@ interface SessionDao {
     @Query("SELECT * FROM session LIMIT 1")
     fun getCurrentSession(): Session?
 
+    @Query("SELECT * FROM session LIMIT 1")
+    fun getCurrentSessionLive(): LiveData<Session?>
+
     @Transaction
     @Query("SELECT * FROM session")
-    fun getSessionWithRegistrations(): List<SessionWithRegistrations>
+    fun getSessionWithRegistrations(): LiveData<SessionWithRegistrations?>
```

Comment:
If there happens to be multiple sessions with registrations, how does this work when we get rid of the result as list? Would it makes sense to have a LIMIT 1 on the query?

Reply:
I'm basing this on the premise that we disallow multiple session with registrations
- we have the UX hard check that won't allow sessions to end until everything is uploaded
- the database primary key is hard coded to 0 (so any attempts to add a new session would throw and exception)

I'm honestly not sure if a limit 1 would limit just the session or the registrations as well. Not entirely sure how the joins work on the back end.

We can try and we'll know if there's a problem (also we can add DB tests)